src/transformState.js

```diff
@@ -5,7 +5,48 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
```

Comment:
Destructure `action`
```suggestion
  for (const { type, extraData, keysToRemove } of actions) {
```